src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -852,6 +852,62 @@ ApiGlobe.prototype.removeEventListenerLayerChanged = function removeEventListene
     this.viewerDiv.removeEventListener('layerchanged:index', this.callbackLayerChanged, false);
 };
 
+/**
+ * Get the IPR (Intellectual Property Rights) of all layers in the scene.
+ * @constructor
+ * @return {array}  An array of IPR.
+ */
+
+ApiGlobe.prototype.getLayersIPR = function getLayersIPR() {
+    var IPR = [];
+    var colorLayers = this.scene.getMap().layersConfiguration.getColorLayers();
+    for (var i = 0; i < colorLayers.length; i++) {
+        IPR.push(colorLayers[i].options.ipr);
+    }
+    var elevationLayers = this.scene.getMap().layersConfiguration.getElevationLayers();
+    for (var j = 0; j < elevationLayers.length; j++) {
+        IPR.push(elevationLayers[j].options.ipr);
+    }
+    IPR = this.removeDuplicatesFromArray(IPR, 'name');
+    return IPR;
+};
```

Comment:
Wasn't exactly what I wanted, I used a Map instead.
I also used `for...of` loop